packages/core/strapi/lib/core/registries/services.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const _ = require('lodash');
+const { pickBy, has } = require('lodash/fp');
+
+const contentTypesRegistry = strapi => {
+  const services = {};
+  const instanciatedServices = {};
+
+  return {
+    get(serviceUID) {
+      if (instanciatedServices[serviceUID]) {
+        return instanciatedServices[serviceUID];
+      }
+
+      const service = services[serviceUID];
+      if (service) {
+        instanciatedServices[serviceUID] = service({ strapi });
+        return instanciatedServices[serviceUID];
+      }
+
+      return undefined;
```

Comment:
maybe throwing would be better suited

Reply:
In the other registres it is like that:
```
    get(policyUID) {
      return policies[policyUID];
    },
```

So returning undefined.
I can throw an error but would need to do it also for the others for consistency

Reply:
Yeah agreed we can do that a bit later :)